scripts/tasks/ts.ts

```diff
@@ -40,10 +44,13 @@ function prepareTsTaskConfig(options: TscTaskOptions) {
     const normalizedOptions = { ...options };
     normalizedOptions.baseUrl = '.';
     normalizedOptions.rootDir = './src';
+    normalizedOptions.project = path.basename(tsConfigFilePath);
```

Comment:
https://github.com/microsoft/just/issues/561

Reply:
Sorry I don't quite understand how the issue relates to this line

Reply:
that we cannot use build mode at this point, which is ok for now.